vendor/src/github.com/docker/libnetwork/endpoint_cnt.go

```diff
@@ -108,6 +108,25 @@ func (ec *endpointCnt) EndpointCnt() uint64 {
 	return ec.Count
 }
 
+func (ec *endpointCnt) updateStore() error {
+retry:
```

Comment:
please, implement a retry logic without a goto.

> This is a common pattern in libnetwork when interacting with the datastore.

That doesn't mean that's a good pattern to follow, and to be honest, it's neither reassuring of good behavior.